urllib3/util/__init__.py

```diff
@@ -6,6 +6,7 @@
 from .ssl_ import (
     SSLContext,
     HAS_SNI,
+    IS_PYOPENSSL,
```

Comment:
Ah, they both seem like feature detection to me. Meh, I don't have strong feelings about this. :)